### ISSUE TITLE
composer.json

### ISSUE THREAD

**Issue by kkrieger85** [campaignmonitor/magento-extension#8]
Please add composer.json file

**Comment by toddhainsworth** [comment]
Agreed, this would be fantastic! While using `package.xml` is perfectly workable, the addition of a `composer.json` would remove some complexity and workflow adjustment. :+1:

**Comment by toddhainsworth** [comment]
@kkrieger85 I've just made a PR for this, feel free to add to it as you see fit!

https://github.com/campaignmonitor/magento-extension/pull/10

**Comment by alexkoepke** [comment]
Hi @DaveOrDead or @nufyoot,

I know @toddhainsworth has already submitted a PR to fix this issue. However, its been open for quite some time and I am hoping this attracts some attention to this issue. 

Also the @toddhainsworth PR could cause some issues because it sets the default Magento dir. However, @toddhainsworth did update the readme.md like a champ!

> ```
>   "extra": {
>     "magento-root-dir": "."
>   }
> ```

I have also submitted PR #11.
